src/com/sudoku/SudokuBoard.java

```diff
@@ -0,0 +1,175 @@
+package com.sudoku;
+
+/**
+ * This class represent the Sudoku board
+ *
+ * @author Alejandra Arteaga
+ * @author Jose Cabrera
+ */
+public class SudokuBoard {
+    private static int BOARD_SIZE = 9;
+    private Cell[][] board = new Cell[BOARD_SIZE][BOARD_SIZE];
+
+    public SudokuBoard(int[][] arrayBoard) {
+        parseToCells(arrayBoard);
+    }
+
+    public int getBoardSize(){
+        return BOARD_SIZE;
+    }
+    public SudokuBoard(Cell[][] board) {
+        this.board = board;
+    }
+
+    /**
+     * Convert an array of integers to an array of Cells
+     *
+     * @param arrayBoard An array of integers
+     */
+    public void parseToCells(int[][] arrayBoard) {
+        for (int row = 0; row < BOARD_SIZE; row++) {
+            for (int column = 0; column < BOARD_SIZE; column++) {
+                this.board[row][column] = new Cell(row, column, arrayBoard[row][column]);
+            }
+        }
+    }
+
+    /**
+     * Get an specific cell from the board
+     *
+     * @param row    The row position in the board
+     * @param column The column position in the board
+     * @return The Cell object
+     */
+    public Cell getCell(int row, int column) {
+        return this.board[row][column];
+    }
+
+    /**
+     * Set a value in an specific cell from the board
+     *
+     * @param row    The row position in the board
+     * @param column The column position in the board
+     */
+    public void setCell(int row, int column, int value) {
+        getCell(row, column).setValue(value);
+    }
+
+    /**
+     * Set a zero value in an specific cell from the board
+     *
+     * @param row    The row position in the board
+     * @param column The column position in the board
+     */
+    public void clearCell(int row, int column) {
+        getCell(row, column).setValue(0);
+    }
+
+    /**
+     * Verify if there is an empty cell in the board
+     *
+     * @return The condition of the board if it contains an empty cell
+     */
+    public Boolean isThereAnEmptyCell() {
+        for (int row = 0; row < BOARD_SIZE; row++) {
```

Comment:
dont you think that for this method you can use the getFirstEmptyCell and if this returns null the board is empty